index.js

```diff
@@ -16,10 +16,19 @@ exports.versionString = function() {
 
 exports.launchServer = function(config) {
   siteBuilder.setConfiguration(config);
+  return webhookValidator.loadKeyDictionary(
+    config.secretKeyFile, config.builders)
+    .then(function(keyDictionary) { return doLaunch(config, keyDictionary); })
+    .catch(function(err) { console.error('Failed to start server:', err); });
```

Comment:
Minor: How about expanding these callbacks to multiple lines?

https://twitter.com/mixonic/status/656935651586256896